ros/angel_system_nodes/angel_system_nodes/berkeley_object_detector.py

```diff
@@ -54,23 +54,27 @@ def __init__(self):
         )
 
         log = self.get_logger()
-        log.info(f"Image topic: {self._image_topic}")
-        log.info(f"Model Config file: {self._model_config}")
-        log.info(f"Detections topic: {self._det_topic}")
-        log.info(f"Detection confidence threshold: {self._det_conf_thresh}")
-        log.info(f"CUDA Device ID: {self._cuda_device_id}")
+        log .info(f"Image topic: {self._image_topic}")
+        log .info(f"Model Config file: {self._model_config}")
+        log .info(f"Detections topic: {self._det_topic}")
+        log .info(f"Detection confidence threshold: {self._det_conf_thresh}")
+        log .info(f"CUDA Device ID: {self._cuda_device_id}")
```

Comment:
These seem to be extraneous spaces between the object and the method invocation. Also for invocations below.